### DISCUSSION TITLE
我的转账丢失了

### DISCUSSION THREAD

**Discussion by TsuiKaKiT** [Chia-Network/chia-blockchain#9165] [Q&A]
我11月1号 发生了一次转账..我查chia浏览器没有到账..但我的xch也扣了
我该怎么办...

会自动退给我吗...已经7天了

**Comment by loppefaaret** [comment]
is the transaction listed in your wallet ? could you perhaps give us the address that was receiving the coins ?

**Reply by TsuiKaKiT** [reply]
收款人地址:  xch14edxaqpwyy2qayavs6fuc8w3waeggfgf2prmluzd3735ykpu0dqqzqxf9a
因为一直没有收到转账 和 退还的关系... 我重新同步了一次钱包... 所以只有一张截图了
![Chia转账截图](https://user-images.githubusercontent.com/93834770/140822069-371b7c77-8e5f-4136-8c5c-29f704481c0e.png)

**Comment by loppefaaret** [comment]
https://www.chiaexplorer.com/blockchain/address/xch14edxaqpwyy2qayavs6fuc8w3waeggfgf2prmluzd3735ykpu0dqqzqxf9a
xch14edxaqpwyy2qayavs6fuc8w3waeggfgf2prmluzd3735ykpu0dqqzqxf9a
4.175145963001 XCH
you say this is the "Payee" address - this address have no outgoing transactions recorded on the blockchain
is the amount on the address correct ? 
this is just 1 single address, your wallet can have many many addresses it controls - so any outgoing transaction might have been taken from another address - do you after having resynced the wallet, believe there is missing XCH ?

**Reply by TsuiKaKiT** [reply]
今天我又同步了一次钱包, 这笔转账的xch 依旧是丢失的...

**Reply by TsuiKaKiT** [reply]
这个收款地址 今天依旧没有没有收到我的转账....

**Reply by loppefaaret** [reply]
Im sorry, but i dont believe im of any good help to you here - you could try our chinese speaking community on our keybase chat:
https://keybase.io/team/chia_network.public
on there, we have a #lang_cn channel for chinese speaking users, i would suggest you try visiting it
